### ISSUE TITLE
Please update disk-trim.md

### ISSUE THREAD

**Issue by noseshimself** [QubesOS/qubes-issues#4473]
### Qubes OS version:

R4.0

### Affected component(s): 

Documentation

---

### Steps to reproduce the behavior:

Read  qubes-doc/configuration/disk-trim.md:
"Add entry to /etc/crypttab (replace luks-<UUID> with the device name and the <UUID> with UUID alone):
luks-<UUID> UUID=<UUID> none discard"

### Expected behavior:

Doesn't work.

### Actual behavior:

Doesn't work.

### General notes:

crypttab's 4th column passes additional args into cryptsetup. The relevant parameter in cryptsetup is "--allow-discards" and there is not parameter "discard"

Please change that line into "allow-discards"

---

**Comment by unman** [comment]
[The documentation](https://www.freedesktop.org/software/systemd/man/crypttab.html) suggests that discard is correct.

> The fourth field, if present, is a comma-delimited list of options. The following options are recognized:
> discard
>     Allow discard requests to be passed through the encrypted block device.

**Comment by noseshimself** [comment]
Fact is, it doesn't work without "allow-discards" for my btrfs root file system; I just tried it on a freshly installed system and got ". And I had the same problem when I installed Qubes 3.2. I'm getting a

sudo fstrim -v /
fstrim: /: the discard operation is not supported

with "discard".

**Comment by marmarek** [comment]
It used to be `allow-discards` but later (systemd 207) was renamed to `discard`: https://github.com/systemd/systemd/blob/7a3bc5a8e52acbaf68cad61aa96b5f7093adc005/NEWS#L6389-L6392
Qubes 3.2 have systemd 222, so after this change.

But, `dracut` (the thing handling initramfs) [was confused about that](https://bugzilla.suse.com/show_bug.cgi?id=932972) and until dracut-044 recognized only `allow-discards`. In Qubes 3.2, there is dracut-043...

In short: for Qubes 3.2 use `allow-discards`, for later versions, use `discard`.

**Comment by unman** [comment]
But @noseshimself is using R4.0

**Comment by tlaurion** [comment]
@andrewdavidwong can be closed.
